gpu-operator/gpu-operator-rdma.rst

```diff
@@ -22,52 +22,80 @@ GPUDirect RDMA and GPUDirect Storage
 About GPUDirect RDMA and GPUDirect Storage
 ******************************************
 
-`GPUDirect RDMA <https://docs.nvidia.com/cuda/gpudirect-rdma/index.html>`_ is a technology in NVIDIA GPUs that enables direct
+`GPUDirect RDMA <https://docs.nvidia.com/cuda/gpudirect-rdma/index.html>`__ is a technology in NVIDIA GPUs that enables direct
 data exchange between GPUs and a third-party peer device using PCI Express. The third-party devices could be network interfaces
 such as NVIDIA ConnectX SmartNICs or BlueField DPUs, or video acquisition adapters.
 
-GPUDirect Storage (`GDS <https://docs.nvidia.com/gpudirect-storage/overview-guide/index.html>`_) enables a direct data path between local or remote storage, like NFS server or NVMe/NVMe over Fabric (NVMe-oF), and GPU memory.
-GDS leverages direct memory access (DMA) transfers between GPU memory and storage, which avoids a bounce buffer through the CPU. This direct path increases system bandwidth and decreases the latency and utilization load on the CPU.
+`GPUDirect Storage <https://docs.nvidia.com/gpudirect-storage/overview-guide/index.html>`__ enables a direct data path between local or remote storage, such as NFS servers or NVMe/NVMe over Fabric (NVMe-oF), and GPU memory.
```

Comment:
Why remove the abbreviation of GPUDirect Storage in the parentheses? Shouldn't we clarify what the abbreviation stands for before referencing it in other places?

Reply:
Fair point.  I made the change because I didn't have an answer to "why start the RDMA paragraph with a link, but use the GDS abbreviation as the link in this paragraph?"  I'll add the abbrev after the link.